panel/config.py

```diff
@@ -650,3 +650,5 @@ def _cleanup_server(server_id):
 panel_extension.add_delete_action(_cleanup_panel)
 if hasattr(panel_extension, 'add_server_delete_action'):
     panel_extension.add_server_delete_action(_cleanup_server)
+
+__all__ = []
```

Comment:
I don't think there is anything you want to publicly expose? So I set `__all__` equal to the empty list. I also do that in other places where I don't think there is anything you want to make a part of the public api to users of the panel package.